.github/dockerfiles/ov_build/fedora_29/Dockerfile

```diff
@@ -38,39 +47,39 @@ RUN cd /usr/src && \
     wget https://www.python.org/ftp/python/3.9.21/Python-3.9.21.tar.xz && \
     tar xvf Python-3.9.21.tar.xz
 RUN cd /usr/src/Python-3.9.21 && \
-    ./configure --enable-optimizations --enable-loadable-sqlite-extensions --prefix=/usr && \
```

Comment:
When I tried rebuilding the image with this option, I encountered several errors in Python like 
```
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
```
w/o this option everything works fine.